src/TransloaditClient.coffee

```diff
@@ -275,15 +272,20 @@ class TransloaditClient
 
   # Wrapper around __remoteJson which will retry in case of error
   _remoteJson: (opts, cb) ->
-    operation = retry.operation(
+    operation = retry.operation
       retries    : 5
       factor     : 3.28
       minTimeout : 1 * 1000
       maxTimeout : 8 * 1000
-    )
 
     operation.attempt =>
       @__remoteJson opts, (err, result) ->
+        if err? && err.error == "RATE_LIMIT_REACHED"
+          console.warn "Rate limit reached, retrying request in #{err.info.retryIn} seconds."
+          # FIXME uses private internals of node-retry
+          operation._timeouts.unshift 1000 * err.info.retryIn
+          return operation.retry err
```

Comment:
Hm, this is a tough one. On the one hand you could use retry with its fairly new `forever` option. I would then add a `stop()` method to it.

The downside of this would be that we'd only have a fixed timeout / set of timeouts, and could not put `err.info.retryIn` to good use. Maybe I should add an `addTimeout()` function to node-retry? What do you think?